src/miniscript/types/mod.rs

```diff
@@ -820,7 +822,7 @@ impl Property for Type {
                 // Note that for CLTV this is a limitation not of Bitcoin but Miniscript. The
                 // number on the stack would be a 5 bytes signed integer but Miniscript's B type
                 // only consumes 4 bytes from the stack.
-                if t == 0 || (t & SEQUENCE_LOCKTIME_DISABLE_FLAG) != 0 {
+                if t.to_consensus_u32() == 0 || (t.to_consensus_u32() & SEQUENCE_LOCKTIME_DISABLE_FLAG) != 0 {
```

Comment:
Perhaps a reason to have `is_zero()` method in `LockTime`? Anyway, this looks nicer to me: `t == LockTime::ZERO`

Also WTF is sequence mixed with lock time here?! Looks very wrong to me!

Reply:
> Anyway, this looks nicer to me: `t == LockTime::ZERO`

:+1:

Reply:
Sure thing.

Reply:
> ecause associated consts don't exist for enums

Did you miss my comment before demonstrating it's not the case?

Reply:
Now includes `LockTime::ZERO` :)